pkg/cmd/pulumi/util.go

```diff
@@ -1146,6 +1146,15 @@ func missingNonInteractiveArg(args ...string) error {
 	}
 }
 
+func promptUserSkippable(yes bool, msg string, options []string, defaultOption string,
+	colorization colors.Colorization,
+) string {
+	if yes {
+		return defaultOption
+	}
+	return promptUser(msg, options, defaultOption, colorization)
+}
```

Comment:
How does this work today? Is this pattern something we should consider moving other uses of `--yes` to?

Reply:
well it does not follow the same interface as `promptForValue`. I did not want to mess around so I decided to create a new function that includes the `yes` parameter.